core/domain/activity_services.py

```diff
@@ -25,6 +25,15 @@
 
 
 def get_featured_activity_references():
+    """
+    Gets a list of featured activity reference models.
+    
+    Args:
+    None
+    
+    Returns:
+    A list having featured activity references.
```

Comment:
Follow the indentation in the [style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments).

Also, specify 'domain objects'. I.e. 
```
    Returns:
        A list of all ActivityReference domain objects that are currently featured.
```